### ISSUE TITLE
Unable to manual select mobile network operator

### ISSUE THREAD

**Issue by zbulb** [PixelExperience/android-issues#5056]
### Build Date

PixelExperience_Plus_munch-13.0-20230422-1331-OFFICIAL

### Device

munch

### Version

thirteen_plus

### Describe the Bug

As per the description, the current release is not able to support the manual selection of mobile network operator

### Steps to Reproduce

1. Under SIM setting > Network, disable Automatically select network
2. Choose network is shown but gives an error message, "Couldn't find networks. Try again."

### Expected Behavior

1. Under SIM setting > Network, disable Automatically select network
2. A discovery of available mobile network operator is carried out and then listed under Choose network
3. Select the mobile network operator the SIM can register to
4. There should be a message indicating the SIM is connected to the selected mobile network operator

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by lastmac** [comment]
I suppose this is the relevant logline from the radio output:
`05-08 14:30:52.695  2824  2824 D RILJ    : [9034]> QUERY_AVAILABLE_NETWORKS  [PHONE1]
05-08 14:30:52.696  2824  3043 D RILJ    : [9034]< QUERY_AVAILABLE_NETWORKS  error 39 [PHONE1]
05-08 14:30:52.696  2824  3043 D RilRequest: [9034]< QUERY_AVAILABLE_NETWORKS  error: com.android.internal.telephony.CommandException: SYSTEM_ERR ret=[] result={ when=-8h5m13s719ms what=51 obj={ when=-8h5m13s719ms what=40 obj=com.android.phone.PhoneInterfaceManager$MainThreadRequest@881e334 target=com.android.phone.PhoneInterfaceManager$MainThreadHandler } target=com.android.internal.telephony.GsmCdmaPhone }`

**Comment by lukasz1992** [comment]
I have Poco F3 which share many drivers, configs and kernel with munch.

I have the same problem and workaround is to switch mode to 4G only (then you could try to switch to 5G again)

**Comment by zbulb** [comment]
Thanks for sharing @lukasz1992 
Your tip does work but still, would be great if it gets fixed proper down the road

**Comment by jp-paper** [comment]
This issue just started to happen, on my Pixel 5 on the latest Android 14, not long after I activated a new eSIM while traveling to a different country.

I found what this person explained here to be very similar to my experience.

https://community.48.ie/t5/Phones/SIM-not-working-on-Android-v14/td-p/60065/page/2

I still haven't figured out how to fix this. Google support teams told me to bring my phone to a service center as they would not be able to diagnose remotely. In the above link, they say that Google asked them to send the device as they had not seen this problem.

On my end, I noticed that I cannot use any secret codes on the phone as even though the SIM cards are detected, they seem to cause issues.

**Comment by jpmn** [comment]
This issue just started to happen, on my Pixel 5 on the latest Android 14, not long after I activated a new eSIM while traveling to a different country.

I found what this person explained here to be very similar to my experience.

https://community.48.ie/t5/Phones/SIM-not-working-on-Android-v14/td-p/60065/page/2

I still haven't figured out how to fix this. Google support teams told me to bring my phone to a service center as they would not be able to diagnose remotely. In the above link, they say that Google asked them to send the device as they had not seen this problem.

On my end, I noticed that I cannot use any secret codes on the phone as even though the SIM cards are detected, they seem to cause issues.
